bedrock/mozorg/templates/mozorg/contribute/thankyou.html

```diff
@@ -47,16 +47,20 @@ <h1 class="page-title">{{ _('Thanks!') }}</h1>
           <span class="profile-greeting">{{ _('Hello!') }}</span>
           <span class="profile-name">{{ _('Your Name') }}</span>
         </div>
-        <h2 class="section-title">{{ _('In the meantime&hellip;') }}</h2>
-        <p class="section-tagline">{{ _('You can create your Mozillians account now to connect with fellow Mozillians.' ) }}</p>
-        <p class="cta-mozillians">
-          {% if settings.DEV %}
-          <a href="https://mozillians-dev.allizom.org/?source=contribute&amp;utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=contribute-page">
+        <h2 class="section-title">{{ _('In the meantime,') }}</h2>
```

Comment:
In the bug user story, the comment on copy is: "In the meantime,  you can help by picking an easy task that only takes a few minutes - or take on a bigger challenge." Not sure if it was intentional to remove the "..." after meantime, but I can confirm.